go.mod

```diff
@@ -167,3 +167,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
Create a types PR once approved.

_Originally posted by @NDStrahilevitz in https://github.com/aquasecurity/tracee/pull/3018#discussion_r1179474297_